README.md

```diff
@@ -2,20 +2,20 @@
 
 PSP22 is a fungible token standard for WebAssembly smart contracts running on blockchains based on the [Substrate][substrate] framework. It is an equivalent of Ethereum's [ERC-20][erc20]. The definition of the PSP22 standard can be found [here][psp22].
 
-This repository contains a simple, minimal implementation of the PSP22 token in [ink!][ink] programming language.
+This repository contains a simple, minimal implementation of the PSP22 token in [ink!][ink] smart contract programming language (EDSL based on Rust).
```

Comment:
What is EDSL?

Reply:
Embedded Domain Specific Language. This is how ink! people call themselves :P